client/components/Admin/Share/ShareList.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { withTranslation, WithTranslation } from 'react-i18next'
-import moment from 'moment'
 import { Table, Alert } from 'reactstrap'
 import Pagination from 'components/Common/Pagination'
 import Crowi from 'client/util/Crowi'
 import { Share } from 'client/types/crowi'
+import { formatToLocaleString } from 'client/util/format-date'
```

Comment:
In our modules, I think file name should be written in camel case.
(I feel this name was affected by `date-fns`.)

Reply:
@lightnet328 
Thank you, I thought that only component files should be written in camel case. (eg: `frontend/crowi.ts`)
If really required, I will fix. :bowing_man:

Reply:
Hmm, sorry it is not clearly written but it is good if `formatDate.ts` in turn with files in `lib/util` 🤔

Reply:
:wrench: changed @ https://github.com/crowi/crowi/pull/594/commits/eafb051aab7f717395eb25089deb011714c903b4